src/frontend/routes/Routes.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import {
+  CartScreen,
+  HomeScreen,
+  ProductsScreen,
+  WishlistScreen,
+} from "../screens";
+import Mockman from "mockman-js";
+import { SignInScreen } from "../screens/SignInScreen/SignInScreen";
+import { SignUpScreen } from "../screens/SignUpScreen/SignUpScreen";
```

Comment:
hey you can import it on lin 3 with other screens 
this might have happened cause of auto import

Reply:
Thanks!
I'll do it.